plotRisk.py

```diff
@@ -1,6 +1,13 @@
 import copy
 
 
+risk = {
```

Comment:
nit: change to `risk_levels`? Idk risk doesn't sound specific enough to me.

Reply:
100%